src/backend/opencl/jit/kernel_generators.hpp

```diff
@@ -28,7 +28,7 @@ void generateParamDeclaration(std::stringstream& kerStream, int id,
 }
 
 /// Calls the setArg function to set the arguments for a kernel call
-int setKernelArguments(
+inline int setKernelArguments(
```

Comment:
will this even be inlined ?

Reply:
Its to avoid unused function warnings in gcc 12.

Reply:
ah new gcc 12!